lib-python/3/sysconfig.py

```diff
@@ -493,9 +493,15 @@ def _init_posix(vars):
     #       while _sysconfigdata_* is a static version of the same values
     #       generated by calling _generate_posix_vars as part of packaging.py.
     #       To prevent chicken-and-egg problems of needing the values before
-    #       generating them, use the dynamic version here.
-    name = os.environ.get('_PYTHON_SYSCONFIGDATA_NAME', '_sysconfigdata')
-    _temp = __import__(name, globals(), locals(), ['build_time_vars'], 0)
+    #       generating them, fall back to the dynamic version here.
+    name = _get_sysconfigdata_name()
+    try:
+        _temp = __import__(name, globals(), locals(), ['build_time_vars'], 0)
+    except ImportError:
+        if '_PYTHON_SYSCONFIGDATA_NAME' in os.environ:
+            raise
+        _temp = __import__("_sysconfigdata", globals(), locals(), ['build_time_vars'], 0)
+
```

Comment:
Maybe we could add a check here that the _temp['INCLUDEPY'] dir exists, and if not use the dynamic one?
```
    else:
        if not os.path.exists(_temp['INCLUDEPY']):
            # Use the dynamic module
            _temp = __import__("_sysconfigdata", globals(), locals(), ['build_time_vars'], 0) 

```